graphics/MoltenVK/Portfile

```diff
@@ -22,29 +23,20 @@ long_description    ${name} is an implementation of the high-performance, \
                     industry-standard Vulkan graphics and compute API, that \
                     runs on Apple's Metal graphics framework.
 
-distname            vulkansdk-macos-${sdkversion}
-use_dmg             yes
+distname            MoltenVK-macos
+use_tar             yes
```

Comment:
They distribute... an uncompressed tar file? What is the matter with these people?